patches/extensions-browser-updater-update_service.cc.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/extensions/browser/updater/update_service.cc b/extensions/browser/updater/update_service.cc
-index 6e96cc25859cb3e84b4a342bd90c3072455ace91..4de3758ac3cb029d5115cb1387c71f1bc8fab492 100644
---- a/extensions/browser/updater/update_service.cc
-+++ b/extensions/browser/updater/update_service.cc
-@@ -35,7 +35,7 @@ namespace {
- UpdateService* update_service_override = nullptr;
- 
- // 98% of update checks have 22 or less extensions.
--constexpr size_t kMaxExtensionsPerUpdate = 22;
-+constexpr size_t kMaxExtensionsPerUpdate = 1;
```

Comment:
I believe this was a privacy issue cc @diracdeltas

Reply:
IIRC we set it to 1 to prevent google from seeing the full set of user extensions, which can be used for fingerprinting.